grace/run.py

```diff
@@ -0,0 +1,86 @@
+from typing import Union
+
+import os
+import click
+import torch
+
+from datetime import datetime
+
+
+from grace.config import write_config_file, load_config_params
+from grace.io.image_dataset import ImageGraphDataset
+from grace.models.train import train_model
+from grace.models.datasets import dataset_from_graph
+from grace.models.classifier import GCN
+from grace.models.feature_extractor import FeatureExtractor
+from grace.utils.transforms import get_transforms
+
+
+@click.command(name="GRACE Trainer")
+@click.option("--config_file", type=click.Path(exists=True))
+def run(config_file: Union[str, os.PathLike]) -> None:
+    """Runs the GRACE pipeline; going straight from images and annotations
```

Comment:
Slight change: going from images and .grace annotations, so it's not confused with annotations in starfiles or other types

Reply:
@mooniean will do!